src/test/scala/org/broadinstitute/dsde/rawls/integrationtest/HttpGoogleCloudStorageDAOSpec.scala

```diff
@@ -59,8 +65,8 @@ class HttpGoogleCloudStorageDAOSpec extends FlatSpec with Matchers with Integrat
     val ownerResource = directory.groups.get(ownerGroup).execute()
 
     // check that the creator is an owner, and that getACL is consistent
-    gcsDAO.getMaximumAccessLevel(testCreator.userEmail, testWorkspaceId) should be (WorkspaceAccessLevel.Owner)
-    gcsDAO.getACL(testWorkspaceId).acl should be (Map(testCreator -> WorkspaceAccessLevel.Owner))
+    Await.result(gcsDAO.getMaximumAccessLevel(testCreator.userEmail, testWorkspaceId), Duration.Inf) should be (WorkspaceAccessLevel.Owner)
```

Comment:
getMaximumAccessLevel now returns a future when it didn't before. In the test we actually want to wait for the result so we can make sure it is correct. These Awaits should only be in test code.